satpy/tests/compositor_tests/test_viirs.py

```diff
@@ -474,6 +478,46 @@ def make_xarray(self, name, calibration, wavelength=None, modifiers=None, resolu
         np.testing.assert_allclose(unique, [24.641586, 50.431692, 69.315375])
 
 
+class ViirsReflectanceCorrectorTest(unittest.TestCase):
```

Comment:
The coverage went down, I think it's because you forgot to include this in the test suite, right ?

Reply:
Oh, good point. Fixed!